src/input/mod.rs

```diff
@@ -287,6 +288,28 @@ impl State {
         Some(pos + target_geo.loc.to_f64())
     }
 
+    fn get_focused_keyboard_shortcuts_inhibitor(&self) -> Option<&KeyboardShortcutsInhibitor> {
+        if let KeyboardFocus::LayerShell { surface }
+        | KeyboardFocus::LockScreen {
+            surface: Some(surface),
+        }
+        | KeyboardFocus::Layout {
+            surface: Some(surface),
+        } = &self.niri.keyboard_focus
```

Comment:
If only there was a convenient `KeyboardFocus::surface()` method that did exactly this :p

Reply:
yeah. if only.